src/libstore/ssh-store.cc

```diff
@@ -24,6 +24,10 @@ struct SSHStoreConfig : virtual RemoteStoreConfig
 class SSHStore : public virtual SSHStoreConfig, public virtual RemoteStore
 {
 public:
+    // Hack for getting remote build log output.
+    // Intentionally not in `SSHStoreConfig` so that it doesn't appear in
+    // the documentation
+    const Setting<int> logFD{(StoreConfig*) this, -1, "log-fd", "file descriptor to which SSH's stderr is connected"};
```

Comment:
N.B. this seemed sketch, but is actually fine because this is exactly how `LegacySSHStore` does it.